vite.config.ts

```diff
@@ -46,6 +46,9 @@ export default ({ command, mode }) => {
         version: "1.3.1",
         whitelistedEnvs: ['VITE_HASURA_GRAPHQL_ENDPOINT', 'VITE_BRANCH', 'VITE_REPOSITORY_URL', 'VITE_SITE_ID', 'VITE_NETLIFY']
       }),
+      replace({
+        'process.env.NODE_ENV': JSON.stringify('development')
```

Comment:
we should not have hardcoded development in case of for example `vite build` - user should decide...